pkgs/by-name/py/pywal16/package.nix

```diff
@@ -3,33 +3,56 @@
   python3,
   fetchFromGitHub,
   imagemagick,
+  installShellFiles,
+  nix-update-script,
 }:
 
 python3.pkgs.buildPythonApplication rec {
   pname = "pywal16";
-  version = "3.6.0";
+  version = "3.7.2";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "eylles";
     repo = "pywal16";
     rev = "refs/tags/${version}";
-    hash = "sha256-YKHOH1bEsZHTgYm8AYpfA6C8RtWxAqNQ+GHMcdaj/JU=";
+    hash = "sha256-aizuON8Y321i7bF2SuC5UC1iOWHY217ccfzY9WmaevQ=";
   };
 
-  nativeBuildInputs = [ python3.pkgs.setuptools ];
+  build-system = [ python3.pkgs.setuptools ];
+
+  nativeBuildInputs = [ installShellFiles ];
 
   nativeCheckInputs = [
     python3.pkgs.pytestCheckHook
     imagemagick
   ];
```

Comment:
```suggestion
    imagemagick
    versionCheckHook
  ];
  versionCheckProgram = "${placeholder "out"}/bin/wal";
  versionCheckProgramArg = [ "-v" ];
```

Reply:
`puwal16` attempts to access the `HOME` environment variable, but `versionCheckHook` runs in a `clean` environment without  it.

> It does so in a clean environment (using env --ignore-environment), and it checks for the ${version} string in both the stdout and the stderr of the command. It will report to you in the build log the output it received and it will fail the build if it failed to find ${version}.

Reply:
Ok, nevermind then. Thanks for trying it though :)